backend/migrations/1656673422615-tasks-effort-and-archive.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class tasksEffortAndArchive1656673422615 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    queryRunner.query(
+      'alter table tasks add column archived boolean not null default true;',
```

Comment:
Although not a problem, it would be good to keep a consistency across the project on the use of uppercase/lowercase. Since the existing migration in the project has the SQL command in uppercase, would you mind using it all in uppercase here as well?